src/components/ArchivedReportFooter.tsx

```diff
@@ -60,7 +53,7 @@ function ArchivedReportFooter({report, reportClosedAction, personalDetails = {}}
     const text = shouldRenderHTML
         ? translate(`reportArchiveReasons.${archiveReason}`, {
               displayName: `<strong>${displayName}</strong>`,
-              oldDisplayName: `<strong>${oldDisplayName}</strong>`,
+              oldDisplayName: `<strong>${oldDisplayName ?? ''}</strong>`,
```

Comment:
I think oldDisplayName can be passed without a fallback empty string.

Reply:
agreed